exercises/tournament/src/test/java/TournamentTest.java

```diff
@@ -1,158 +1,160 @@
-import org.junit.Rule;
+import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.rules.ExpectedException;
 
-import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
 
 public class TournamentTest {
 
-    @Rule
-    public ExpectedException expectedException = ExpectedException.none();
+    private Tournament tournament;
+
+    @Before
+    public void setUp() throws Exception {
+        tournament = new Tournament();
+    }
 
     @Test
     public void justTheHeaderIfNoInput() {
-        Tournament tournament = new Tournament();
-        assertEquals("Team                           | MP |  W |  D |  L |  P", tournament.printTable());
+        assertEquals("Team                           | MP |  W |  D |  L |  P\n", tournament.printTable());
     }
 
     @Test
-    public void aWinIsThreePointsALossIsZeroPoints() throws Exception {
-        Tournament tournament = new Tournament();
+    @Ignore
+    public void aWinIsThreePointsALossIsZeroPoints() {
         tournament.applyResults("Allegoric Alaskans;Blithering Badgers;win");
-        assertArrayEquals(new String[]{
-                "Team                           | MP |  W |  D |  L |  P",
-                "Allegoric Alaskans             |  1 |  1 |  0 |  0 |  3",
-                "Blithering Badgers             |  1 |  0 |  0 |  1 |  0"
-                }, tournament.printTable().split("\n"));
+        assertEquals(
+                "Team                           | MP |  W |  D |  L |  P\n" +
+                "Allegoric Alaskans             |  1 |  1 |  0 |  0 |  3\n" +
+                "Blithering Badgers             |  1 |  0 |  0 |  1 |  0\n"
+                , tournament.printTable());
```

Comment:
The indentation and position of the comma here and in later tests is a bit inconsistent. The comma should come after the first parameter (on the same line) and the second parameter should be indented four spaces in from `assertEquals`. I would indent it like this;
```
 assertEquals(
    "Team                           | MP |  W |  D |  L |  P\n" +
    "Allegoric Alaskans             |  1 |  1 |  0 |  0 |  3\n" +
    "Blithering Badgers             |  1 |  0 |  0 |  1 |  0\n" ,
    tournament.printTable()
);

I also see now that the canonical data has the output as an array (sorry, didn't look at it closely enough before). I think it's fine to leave your output as a string if you prefer, it shouldn't matter too much. Or you can have it as an array, but in that case I think the `printTable` method should return an array, it shouldn't be converted to an array in the tests. Does that make sense? Do you agree? But what you have at the moment it fine as well :)